examples/features/customloadbalancer/client/customroundrobin/customroundrobin.go

```diff
@@ -31,8 +31,11 @@ import (
 	"google.golang.org/grpc/serviceconfig"
 )
 
+var gracefulSwitchPickFirst serviceconfig.LoadBalancingConfig
+
 func init() {
 	balancer.Register(customRoundRobinBuilder{})
+	gracefulSwitchPickFirst, _ = endpointsharding.ParseConfig(json.RawMessage(endpointsharding.PickFirstConfig))
```

Comment:
Please add `err` capturing and make sure no error was returned, or `logger.Fatal` if so.

Reply:
Done.